python/paddle/fluid/reader.py

```diff
@@ -1117,7 +1118,7 @@ def reset(self):
 
     @classmethod
     def _check_input_array(cls, item):
-        arr = np.array(item)
+        arr = np.asarray(item)
```

Comment:
Duplicate `_check_input_array`?

Reply:
Done. Move _check_input_array  to DataLoaderBase.